docs/commands/generate-codegen.md

```
@@ -3,7 +3,7 @@
 > #### Create a library based on the OpenAPI specification. If the openapi specification has not yet been generated, this will generate that as well as the first step. Supports **all** openapi tools generators + orval clients.
 
 ```term
-$ xano generate codegen [options]
+$ caly-xano generate codegen [options]
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)